python/lsst/cp/pipe/ptc/measurePtcGen2Task.py

```diff
@@ -124,7 +124,7 @@ def runDataRef(self, dataRefList):
                 self.log.warn("postISR exposure could not be retrieved. Ignoring flat.")
                 continue
             expList.append(tempFlat)
-        expIds = [exp.getInfo().getVisitInfo().getExposureId() for exp in expList]
+        expIds = [exp.info.getVisitInfo().id for exp in expList]
```

Comment:
Should this be `exp.info.id for exp in expList`?

Reply:
Oops, I could swear I included an explanation in the commit message (I'll go back and add one).

@czwa told me that these two identifiers were supposed to be the visit ID and not the per-detector ID, so basically the use of `getExposureId()` here was a bug.